quarkchain/cluster/p2p_commands.py

```diff
@@ -197,11 +197,22 @@ def __init__(self, block):
         self.block = block
 
 
+class PingPongCommand(Serializable):
+    """
+    with 32B message which is undefined at the moment
+    """
+
+    FIELDS = [("message", hash256)]
+
+    def __init__(self, message):
+        self.message = message
+
+
 class CommandOp:
     HELLO = 0
     NEW_MINOR_BLOCK_HEADER_LIST = 1
     NEW_TRANSACTION_LIST = 2
-    GET_PEER_LIST_REQUEST = 3
+    GET_PEER_LIST_REQUEST = 3  # only handled by simple_network peers
```

Comment:
what is the purpose of enabling this command?

Reply:
simple_network uses this for trivial discovery, I double-checked that this is not handled for real p2p network